### PR TITLE
remove reference to docker_posix.json

### PR DESCRIPTION
**Pull request by jmaher**

### CODE REVIEW

File: tests/fixtures/no_tasks_on_condition.taskcluster.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ tasks:
           image: "rust:buster"
           env:
             CODECOV_TOKEN: 4df01912-087e-489a-be28-25aa911cb9d2
-            CARGO_INCREMENTAL: 0
```
Comment (marco-c) [inline tests/fixtures/no_tasks_on_condition.taskcluster.yml:69]: Why is this needed?

Comment (jmaher) [reply tests/fixtures/no_tasks_on_condition.taskcluster.yml:69]: a failure in validator: https://community-tc.services.mozilla.com/tasks/Ts7olSbCQ-iTU0crfulEaQ/runs/0/logs/public/logs/live.log

**Review by marco-c — APPROVED** [review]
